numpy/polynomial/_polybase.py

```diff
@@ -284,9 +284,41 @@ def __repr__(self):
         return f"{name}({coef}, domain={domain}, window={window})"
 
     def __str__(self):
-        coef = str(self.coef)
-        name = self.nickname
-        return f"{name}({coef})"
+        # Get configuration for line breaks
+        linewidth = np.get_printoptions().get('linewidth', 75)
+        if linewidth < 1:
+            linewidth = 1
+        out = ""
+        if self.coef[0] < 0:
+            out += "-"
+        out = f"{self.coef[0]} "
+        cur_line = len(out) // linewidth    # For handling linebreaks
+        for i, coef in enumerate(self.coef[1:]):
+            next_term = ""
+            power = str(i + 1)
+            # Polynomial coefficient
+            if coef >= 0:
+                next_term += f"+ {coef}"
+            else:
+                next_term += f"- {np.abs(coef)}"
+            # Polynomial term
+            next_term += self._str_term(power, "x")
+            # Handle linebreaks
+            if (len(out) + len(next_term)) // linewidth != cur_line:
+                next_term = next_term.replace(" ", "\n", 1)
```

Comment:
not sure if this will be interpreted as newline on windows or if it expects "\r\n"

Reply:
I'm not sure - the test suite, which tests multiline printing, passes on the CI (I've reverted back to 7940d81f6 so that that the code is back in sync with the tests). Hopefully that is sufficient to indicate there aren't any problems expected w.r.t. newline characters.

Reply:
This is handled by the stdout stream itself, no python code working with strings usually even needs to ever worry about this.